README.md

```diff
@@ -56,34 +56,22 @@ If you don't want the whole framework, you can just `@include` specific [parts](
 
 ## Vanilla local development
 
-To develop on Vanilla itself, simply pull down the project and make changes.
-
-To test it locally using our recommended tooling, you need to first install dependencies:
-
-You need to have bundler installed:
-
-``` bash
-gem install bundler
-```
-
-You need to have node and npm. To install follow the [installation instructions](https://docs.npmjs.com/getting-started/installing-node) on this page.
+The simplest way to run Vanilla framework is to first [install Docker](https://docs.docker.com/engine/installation/), add your user to the `docker` group, and then use the `./run` script:
```

Comment:
The problem is that I don't want to get into the nitty gritty of how to manage specific operating systems, because that can get really long and really messy, and has nothing to do with this codebase or its direct dependencies. I'd rather leave it to Docker/the internet at large to document how you set it up correctly on different OSes.

What we could do is link to instructions for adding the user to group for different OSes as simply as possible. However, I think maybe this step is no longer necessary for macOs. If we can verify that than we can say:

> [on Linux, add your user to the `docker` group](https://docs.docker.com/engine/installation/linux/linux-postinstall/)